capitalized.txt

```diff
@@ -0,0 +1,3 @@
+"PIB WDMCZPU"
+
+LOLOLOLOLOLOLOLOLOL
```

Comment:
Tested this with simple text to take on the an unfamiliar concept in smaller chunks.